src/methods/partial_randomization.jl

```diff
@@ -2,19 +2,21 @@ export PartialRandomization, PartialRandomizationAAFT
 
 """
     PartialRandomization(α = 0.5)
-`PartialRandomization` surrogates[^Ortega1998] are similar to [`RandomFourier`](@ref) phase 
+`PartialRandomization` surrogates[^Ortega1998] are similar to [`RandomFourier`](@ref) phase
 surrogates, but during the phase randomization step, instead of drawing phases from `[0, 2π]`,
-phases are drawn from `[0, 2π]*α`, where `α ∈ [0, 1]`. The authors refers to `α` as the 
-"degree" of phase randomization, where `α = 0` means `0 %` randomization and 
+phases are drawn from `[0, 2π]*α`, where `α ∈ [0, 1]`. The authors refers to `α` as the
+"degree" of phase randomization, where `α = 0` means `0 %` randomization and
 `α = 1` means `100 %` randomization.
 
+See [`RelativePartialRandomization`](@ref) and [`SpectralPartialRandomization`](@ref) for alternative partial-randomization algorithms.
```

Comment:
Most of our code is formatted using a maximum line length of 92 characters. It would be nice if this PR follows that too, for example on this line! (don't worry about the line length for citations, we'll move to DocumenterCitations.jl for those soon).